src/ComputeSharp.D2D1/Attributes/Enums/D2D1BufferPrecision.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace ComputeSharp.D2D1;
+
+/// <summary>
+/// Indicates the bit depth to use in the imaging pipeline in Direct2D to produce the result of a D2D1 shader.
+/// </summary>
+/// <remarks>
+/// <para>This type exposes the available values in <see href="https://docs.microsoft.com/windows/win32/api/d2d1_1/ne-d2d1_1-d2d1_buffer_precision"/>.</para>
+/// <para>For more info, see <see href="https://docs.microsoft.com/windows/win32/direct2d/precision-and-clipping-in-effect-graphs"/>.</para>
+/// </remarks>
+public enum D2D1BufferPrecision
+{
+    /// <summary>
+    /// The buffer precision is not specified.
+    /// </summary>
+    Unknown,
```

Comment:
can you make these explicitly equal to the D2D constants?

e.g.
```cs
Unknown = D2D1_BUFFER_PRECISION_UNKNOWN,
Int8Normalized = D2D1_BUFFER_PRECISION_8BPC_UNORM,
```

I'd like to be able to just cast to my enum value, which is defined this way
<img width="336" alt="image" src="https://user-images.githubusercontent.com/10873410/167271616-f4cc242e-b272-4d5a-8bab-607b3686277c.png">

Reply:
Fixed in d6753ecaed2b9ae5d99a6c8c748ca18d00de3e8c.